colabs/raytune/W&B + RayTune.ipynb

```diff
@@ -0,0 +1,147 @@
+{
```

Comment:
<p>Line 4: Love the inclusion of a docstring. The note about <code>wandb.log</code> being allowed only within its scope is great, but I think that should really be in the body of the text outside the cells, as you describe what the mixin is for.</p><p> </p><p>Line 7-8: I might move the imports of the example utilities down from the top of the notebook and closer to where they get used. Specifically note that you're using them, maybe mention that they're useful for debugging, link to any documentation.</p><p> </p><p>Line 10: put a space after the comma between <code>["lr"]</code> and <code>momentum</code></p><p> </p><p>Line 12: I like that you've fixed the number of iterations to be 10 to keep runtime down!</p><p> </p><p>Line 13: put a space after the comma between <code>train_loader</code> and <code>device</code></p><p> </p><p>Line 14: put a space after the comma between <code>test_loader</code> and <code>device</code></p><p> </p><p>Line 16: I suggest using "Error Rate" as the custom metric: <code>100 * (1 - acc)</code>. It's got a meaning, but it's still trivial to calculate. You'd update the name in Line 18.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/37/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>